configs/sst_kernel_ft-rasdaemon.yaml

```diff
@@ -18,3 +18,16 @@ data:
   #           - arch-specific-package1   # <-- The list of binary RPM packages
   #               - arch-specific-package1   #     goes here.
   #
+  arch_packages:
+      x86_64:
+          - rasdaemon
```

Comment:
I see you're listing `rasdaemon` as an arch-specific package, but it's also in the `packages` section which would pull in into all arches again.

You might might want to use:

```
packages: []
```

Reply:
That was my guess to fix:
https://tiny.distro.builders/workload-overview--sst_kernel_ft-rasdaemon--repository-fedora-eln.html

Apparently ExclusiveArch is being ignored and it's being tested in s390.